test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -93,8 +103,37 @@
 
     (is (expected-https-config?
           (merge valid-ssl-pem-config
-                 {:ssl-host "foo.local" :ssl-port 8001 :request-header-max-size 16192})
-          {:https {:host "foo.local" :port 8001 :request-header-max-size 16192}})))
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :request-header-max-size 16192})
+          {:https {:host "foo.local"
+                   :port 8001
+                   :request-header-max-size 16192}}))
+
+    (is (expected-https-config?
+          (merge valid-ssl-pem-config
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :so-linger-seconds 22})
```

Comment:
I tried setting this to a float, and got an exception.  Since the underlying socket option takes a millisecond value, I think it'd be best to support accepting floating point values so sub-second timeouts can be configured if necessary.